### PR TITLE
Translating reference react dom.md

### PR DESCRIPTION
**Pull request by splimter**

### CODE REVIEW

File: content/docs/reference-react-dom.md [hunk 1]
```diff
@@ -1,114 +1,113 @@
...
+- [`()hydrate`](#hydrate)
+- [`()unmountComponentAtNode`](#unmountcomponentatnode)
+- [`()findDOMNode`](#finddomnode)
+- [`()createPortal`](#createportal)
 
-### ??? ??????? {#browser-support}
+### دعم المتصفح {#browser-support}
 
-???? React ???? ????????? ???????? ??? ?? ??? Internet Explorer 9 ??? ???? ??? ?????? ??? ????? ??? ????????? ??????? ??? Internet Explorer 9 ? Internet Explorer 10.
+تدعم React جميع المتصفحات الشائعة، بما في ذلك Internet Explorer 9 فما فوق، رغم الحاجة إلى توفير دعم للمتصفحات القديمة مثل Internet Explorer 9 و Internet Explorer 10.
```
Comment (iRayan7) [inline content/docs/reference-react-dom.md]: Original line "React supports all popular browsers, including Internet Explorer 9 and above, although [some polyfills are required](/docs/javascript-environment-requirements.html) for older browsers such as IE 9 and IE 10."

File: content/docs/reference-react-dom.md [hunk 2]
```diff
@@ -1,114 +1,113 @@
...
+إن كان عنصر React مُصيَّر سابقًا إلى الحاوية`container`, فسيُجري تحديثًا عليه ويُعدِّل DOM فقط كما هو ضروري ليعكس آخر تحديثات عنصر React.
 
-?? ????? ?? ?????? ????????? ????????? ??? ????? ?? ????? ???????.
+إن أضفنا رد النداء الاختياري فسيُنفَّذ بعد تصيير أو تحديث المكوّن.
 
-> ??????:
+> ملاحظة:
 >
-> ?????? ??????`ReactDOM.render()` ???????? ?????? ??????? ???? ?????????. ???????? ?? ????? ?????? ??????? ??? ??? ???????. ?????? ??????????? ??????? ???????? ???????? ?? ??? ??????? ??????.
+> يتحكّم التابع `ReactDOM.render()` بمحتويات العقدة الحاوية التي تُمرِّرها. تُستبدَل أي عناصر موجودة بداخلها عند أول استدعاء. تستخدم الاستدعاءات اللاحقة خوارزمية المقارنة من أجل التحديث بكفاءة.
```
Comment (iRayan7) [inline content/docs/reference-react-dom.md]: container node (العقدة الحاوية)

File: content/docs/reference-react-dom.md [hunk 3]
```diff
@@ -1,114 +1,113 @@
...
 * * *
 
-### `findDOMNode()` {#finddomnode}
+### `()findDOMNode` {#finddomnode}
 
-> ??????:
+> ملاحظة:
 >
-> ?????? `findDOMNode` ?? ????? ???? ??????? ?????? ??? ???? DOM ???????. ?? ??? ?????? ?? ???? ??????? ??????? ????? ?????? ??? ?????? ???? ????? ?????????. [??? ???? ?? ????? ??????.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
+> التابع `findDOMNode` و وسيلة هروب مستخدمة للوصول إلى عقدة DOM التحتية. من غير المفضل في معظم الأحيان استخدام وسيلة الهروب هذه لأنّها تخرق تجريد المكوّنات. لقد . [أهمل في الوضع الصارم `StrictMode`.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
```
Comment (iRayan7) [inline content/docs/reference-react-dom.md]: Delete `.` after `لقد`

File: content/docs/reference-react-dom.md (hunk 2 above)
Comment (splimter) [reply content/docs/reference-react-dom.md]: would u like to use the container node instead of العقدة الحاوية ??

Comment (iRayan7) [reply content/docs/reference-react-dom.md]: use them together "العقدة الحاوية (container node)"

Comment (splimter) [reply content/docs/reference-react-dom.md]: Done

**Pull request merged**